modules/indexing/src/main/java/org/apache/ignite/internal/processors/query/stat/IgniteStatisticsConfigurationManager.java

```diff
@@ -667,6 +716,12 @@ private void onChangeStatisticConfiguration(
         StatisticsObjectConfiguration oldCfg,
         StatisticsObjectConfiguration newCfg
     ) {
+        if (!active) {
```

Comment:
should we check the state inside the synchronized block below?

Reply:
State check removed.